server/controllers/chartsController.js

```diff
@@ -104,6 +107,12 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
           targetTotal: newTargetValue,
         }
       }
+      studyTotals[constants.TOTALS_STUDY].targetTotal =
+        !!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal
+          ? studyTotals[constants.TOTALS_STUDY].targetTotal + newTargetValue
+          : !!newTargetValue && !studyTotals[constants.TOTALS_STUDY].targetTotal
+          ? newTargetValue
+          : studyTotals[constants.TOTALS_STUDY].targetTotal
```

Comment:
🧑‍🎓 what does it mean to have a `targetTotal` equal to `undefined` instead of `0`?

Reply:
undefined means it was explicitly not set. The requirements are that is a site does not have a target value set for a variable then we want to use the total count of subjects to calculate percentages. If all sites have a target value set for a variable then we want to use the target count of subjects (target value) to calculate percentages.